charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```diff
@@ -9,7 +9,7 @@ metadata:
     {{- end }}
   {{- end }}
   name: {{ template "argo-cd.controller.fullname" . }}
-  namespace: {{ .Release.Namespace | quote }}
+  namespace: {{ include "argo-cd.namespace" . | quote }}
```

Comment:
One small nit. Do we need `quote` here? We don't have it anyway in RBAC bindings.